repo2docker/buildpacks/r.py

```diff
@@ -186,6 +188,29 @@ def get_packages(self):
 
         return super().get_packages().union(packages)
 
+    def _get_latest_working_mran_date(self, startdate, max_prior):
+        """
+        Look for a working MRAN snapshot
+
+        Starts from `startdate` and tries up to `max_prior` previous days.
+        Raises `requests.HTTPError` with the last tried URL if no working snapshot found.
+        """
+        for days in range(0, max_prior + 1):
```

Comment:
Why explicitly set the start of `range` to zero? As it is the default value I was expecting `range(max_prior + 1)`